misc/emacs/mgmtconfig-mode.el

```diff
@@ -0,0 +1,90 @@
+;;; mgmtconfig-mode.el --- mgmt configuration management language
+
+;; Copyright 2017 Peter Oliver.
```

Comment:
Sort of an annoying nitpick, but do you mind changing the copyright notice matches the exact text as shown here:

https://github.com/purpleidea/mgmt/blob/master/main.go#L1

This would just help keep things legally consistent, and in no way takes away from your copyright or attribution.

You can keep the keywords/commentary stuff at the bottom, and I would recommend leaving it to `git` instead of the "authors" part, but it's up to you. It's nicer if we get many contributors to this file to avoid a long string of names here...

Reply:
The keywords/commentary/etc. metadata gets passed through to the package browser UI within Emacs.  I agree that providing an author's name and email address seems anachronistic in these days of ubiquitous public version control systems, but Emacs people can be very old fashioned.